### DISCUSSION TITLE
Increasing the email speed of the worker

### DISCUSSION THREAD

**Discussion by BlackIsBlack** [fief-dev/fief#417] [Q&A]
How can I increase the speed of the emails sent by the worker? Sometimes they can take multiple minutes to send, and this is causing a point of friction between end users, as sometimes they have to wait 4-5 minutes for a verification email to be sent?

Is there a param that I can setup in the worker, or will creating more worker containers solve this issue?

**Comment by frankie567** [comment]
Are you sure it comes from the worker? Normally, it should handle the incoming jobs in a matter of seconds. However, depending on your email provider, the email might take some time to arrive in the user's inbox; but that's out of Fief's control.

To make sure of it, look at the worker logs. You should look for a line `Done task task=on_email_verification_requested`.
